plugins/statistics.koplugin/main.lua

```diff
@@ -486,6 +490,7 @@ function ReaderStatistics:saveSettings(fields)
 end
 
 function ReaderStatistics:onReadSettings(config)
+    -- delay initialization for accurate total pages count of the doc
```

Comment:
I tested and confirm that crengine will report the correct page count after the `:render()` call. I added an event in https://github.com/koreader/koreader/pull/1798 to help avoid using `0.1` with schedule. `0.1` is a magic number that makes the code less readable. I have seen multiple uses of this pattern in our code base.

I think we are just using it as the `process.nextTick` call in nodejs. We should either implement our own `nextTick` helper function or replace those `0.1` schedule call with a proper event. A proper event will be the ideal approach sine it makes the code deterministic while the behavior of `nextTick` will depend on the scheduler.